### PR TITLE
[Keyboard] Add converter/xmk

### PR DESCRIPTION
**Pull request by manna-harbour**
## Description

This PR adds new keyboard `converter/xmk`. This is the QMK keyboard definition for [𝑥MK](https://github.com/manna-harbour/xmk). Also see https://github.com/manna-harbour/xmk/discussions/2.

Thanks!

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/xmk/readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!--
+Copyright 2022 Manna Harbour (@manna-harbour)
```
Comment (drashna) [inline keyboards/converter/xmk/readme.md:2]: License header isn't needed here.

File: keyboards/converter/xmk/readme.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+https://github.com/manna-harbour/xmk
+
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# converter/xmk
+
+![𝑥MK](https://raw.githubusercontent.com/manna-harbour/xmk/main/docs/images/xmk-banner.jpg)
```
Comment (drashna) [inline keyboards/converter/xmk/readme.md:10]: Please rehost on Imgur.

File: keyboards/converter/xmk/readme.md (hunk 1 above)
Comment (manna-harbour) [reply keyboards/converter/xmk/readme.md]: Removed.

File: keyboards/converter/xmk/readme.md (hunk 2 above)
Comment (manna-harbour) [reply keyboards/converter/xmk/readme.md]: Removed.

**Comment by manna-harbour** [comment]
@drashna Thanks for the review!

**Comment by manna-harbour** [comment]
The requested changes have been made plus an extra small fix and additional feature, and this is again ready for review. Thanks!

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Comment by manna-harbour** [comment]
Thanks everyone!

**Pull request merged**